README.md

```diff
@@ -45,18 +45,19 @@ While you can also control the low-level driver manually:
 ```rust,no_run
 use compio::{
     buf::IntoInner,
-    driver::{AsRawFd, Driver, Poller},
+    driver::{AsRawFd, AsRegisteredFd, Driver, Poller, RegisteredFileDescriptors},
```

Comment:
I think we need to talk about the name "registered". It's too long. Maybe `RegFd` is OK.